### DISCUSSION TITLE
-f=text for transcripts without timestamps doesn't work

### DISCUSSION THREAD

**Discussion by Salsero69906** [Purfview/whisper-standalone-win#273] [Q&A] [answered]
The transcriptions are great, but I need transcripts without timestamps. Using “-f=text” as suggested here (https://github.com/Purfview/whisper-standalone-win/discussions/112) or “--output_format=text“  does not work for me. Transcripts still have timestamps at every line (unsing models medium and medium.en).

.\Faster-Whisper-XXL\faster-whisper-xxl.exe "C:\users\[username]\documents\S3.mp3" --language=en --model=medium --output_format=text

I had also tried “--without_timestamps=true” and “--word_timestamps=false”. While the latter produces less timestamps, they still appear at the beginning of each 30s chunk, and it often cuts sentences in the middle.

Can someone help? Thank you.

**Comment by Purfview** [comment, marked as answer]
It works, look at the output file, not at the console output.

**Reply by Salsero69906** [reply]
Oh! You are right! Sorry about this. Since the console output shows the timestamps, I didn't check the output file.
Thank you for your very swift and helpful response!
